experiments_main.cpp

```diff
@@ -19,9 +19,10 @@
 DEFINE_uint32(kmer_size, 3, "Kmer size for MH, OMH, WMH");
 DEFINE_uint32(k, 3, "Short hand for --kmer_size");
 
-DEFINE_string(alphabet,
-              "dna4",
-              "The alphabet over which sequences are defined (dna4, dna5, protein)");
+//DEFINE_string(alphabet,
+//              "dna4",
+//              "The alphabet over which sequences are defined (dna4, dna5, protein)");
+DEFINE_int32(alphabet_size, 4, "size of alphabet for synthetic sequence generation");
```

Comment:
What's the reason for this change? Currently TS supports 3 alphabets (dna4, dna5 and protein). If you'd like to add more alphabets, you can add them in alphabets.cpp. But I see that you are still using an alphabet size of 4, so I see no reason for this change.

Reply:
experiments_main.cpp assumes that if alphabet_size is set to X, the characters will be in the [0,...,X-1] range, but in the alphabets.cpp, for both dna4 and dna5, the alphabet are in the [0,..,X] range

Reply:
this unnecessarily limits the alphabet_size of the synthetic sequence generation to 4 or 5. For sake of the experiments only, the alhpabet_size is sufficient and gives more flexibility